resolwe_bio/descriptors/bbduk-star-htseq.yml

```diff
@@ -1,17 +1,28 @@
 - slug: quantseq
-  name: QuantSeq workflow
-  version: 0.0.2
+  name: Pipeline for reads prepared with FWD or REV QuantSeq Kit by Lexogen
+  version: 0.0.3
   description: QuantSeq workflow selection template
   schema:
     - name: stranded
-      label: Were samples processed with QuantSeq FWD or REV Library?
+      label: Select the QuantSeq kit used for library preparation.
```

Comment:
Don't put the whole sentence in the label. Consider just _QuantSeq kit used for library preparation_ (no period at the end). Add a description attribute, if more explanation is needed.

Reply:
This was defined in Spec: https://docs.google.com/document/d/1CXRdEmsFm159jzl1ONXZkTQmMetCAmFxXXlokcgYfsU/edit#heading=h.m4r3szh258tw